src/com/tutorial/main/Game.java

```diff
@@ -1,134 +0,0 @@
-package com.tutorial.main;
-
-import java.awt.Canvas;
-import java.awt.Color;
-import java.awt.Graphics;
-import java.awt.image.BufferStrategy;
-import java.util.Random;
-
-public class Game extends Canvas implements Runnable{
-
-	/**
-	 * 
-	 */
-	private static final long serialVersionUID = 1L;
```

Comment:
This is useless. `serialVersionUID` is used for serialized objects (`...implements Serializable`) and is generally only used for web services anymore.